strapi/strapi
# [Upgrade tool] Update project package.json with target version

### What does it do?

When an upgrade command is run, modify the project's package.json with the target version of Strapi.

### Why is it needed?

To facilitate upgrading the version of a Strapi project

### How to test it?

Run the upgrade tool and see that the package.json file for the project is upgraded.

Tests will be written before the final feature is merged.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
